GVRf/Framework/src/org/gearvrf/scene_objects/GVRTextViewSceneObject.java

```diff
@@ -103,6 +128,9 @@ public GVRTextViewSceneObject(GVRContext gvrContext, GVRMesh mesh,
         mSurface = new Surface(mSurfaceTexture);
         mSurfaceTexture.setDefaultBufferSize(mTextViewContainer.getWidth(),
                 mTextViewContainer.getHeight());
+        sReferenceCounter++;
```

Comment:
Is this static variable intended to serialize drawing text frames one at a time for the purpose of performance load balancing? If so, it would be helpful to add some in source documentation for future reference.